src/libraries/System.ComponentModel.Annotations/src/System/ComponentModel/DataAnnotations/Validator.cs

```diff
@@ -190,6 +188,8 @@ public static class Validator
         public static bool TryValidateValue(object value, ValidationContext validationContext,
             ICollection<ValidationResult>? validationResults, IEnumerable<ValidationAttribute> validationAttributes)
         {
+            ArgumentNullException.ThrowIfNull(validationAttributes, nameof(validationAttributes));
```

Comment:
```suggestion
            ArgumentNullException.ThrowIfNull(validationAttributes);
```

Reply:
I didn't know if this worked yet or not. If it does, I'll change to this.